sdk/eventhub/eventhubs-checkpointstore-tables/test/tables-checkpointstorespec.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+
+import { TableCheckpointStore } from "../src";
+
+
```

Comment:
There's more whitespace than I expected. Have you run `npm run format` on the project?